sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -670,11 +670,12 @@ export const PendingDesignationsHeaders = [
 export const OfferingSummaryHeaders = [
   { title: "Name", sortable: true, key: "name" },
   { title: "Year of Study", sortable: false, key: "yearOfStudy" },
-  { title: "Study Dates", sortable: false, key: "studyDates" },
-  { title: "Intensity", sortable: false, key: "offeringIntensity" },
+  { title: "Study Start Date", sortable: true, key: "studyStartDate" },
+  { title: "Study End Date", sortable: true, key: "studyEndDate" },
+  { title: "Intensity", sortable: true, key: "offeringIntensity" },
   { title: "Offering type", sortable: false, key: "offeringType" },
-  { title: "Study delivery", sortable: false, key: "offeringDelivered" },
-  { title: "Status", sortable: false, key: "offeringStatus" },
+  { title: "Study delivery", sortable: true, key: "offeringDelivered" },
+  { title: "Status", sortable: true, key: "offeringStatus" },
```

Comment:
The consistency is missing in following the upper case here. 

For e.g. Study Start Date has all words starting in uppercase where Offering type and Study delivery isn't